python/lsst/ip/isr/isrTask.py

```diff
@@ -1147,15 +1147,15 @@ def readIsrData(self, dataRef, rawExposure):
             except NoResults:
                 try:  # Fall back to the old-style numpy-ndarray style kernel if necessary.
                     brighterFatterKernel = dataRef.get("bfKernel")
-                    self.log.info("Old style brighter-fatter kernel (np.array) loaded")
+                    self.log.info("Old style brighter-fatter kernel (bfKernel) loaded")
                 except NoResults:
                     brighterFatterKernel = None
             if brighterFatterKernel is not None and not isinstance(brighterFatterKernel, numpy.ndarray):
```

Comment:
All of the changes (except for the log message above) is protected by this if check.  This means if we received a numpy array (which is the case for HSC), then this section is skipped.  All of the other changes to support new brighter-fatter kernels has been done in gen3, which uses the `runQuantum` method to check and restructure calibrations.  When we switch fully to gen3, this method will be removed, as it's only called in gen2.

Reply:
Ah, that's what I was missing...thanks for the explanation!